### PR TITLE
Support/europa enablers

### PR DESCRIPTION
**Pull request by ofreyssinet-ledger**
### 📝 Description

- New feature flags, removal of `syncOnboarding` feature flag:
  - In device selection flows, Stax (resp. Europa) device should be there if and only if the feature flag `supportDeviceStax` (resp. `supportDeviceEuropa`) is enabled. Previously that was conditioned by the `syncOnboarding` flag.
- Custom lock screen `customImage` feature flag removal.
  - In "My Ledger", the "lockscreen" CTA should be visible if and only if the device is a Stax or Europa.
  - On NFTs, the option "Set as Stax lockscreen" should be visible if and only if a Stax or Europa device has been previously connected to this install of Ledger Live. This state of Ledger Live can be modified by going to Debug settings > Configuration > "Has connected once to ..."
- Some placeholder illustrations have been added for Europa, but the lotties are not updated for now. 

### ❓ Context

- **JIRA or GitHub link**: [LIVE-11306] [LIVE-11365] [LIVE-11366] [LIVE-11367]

### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [x] `npx changeset` was attached.
- [x] **Covered by automatic tests.** 
- [x] **Impact of the changes:** 
  - cf. description and changesets

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-11306]: https://ledgerhq.atlassian.net/browse/LIVE-11306?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[LIVE-11365]: https://ledgerhq.atlassian.net/browse/LIVE-11365
[LIVE-11366]: https://ledgerhq.atlassian.net/browse/LIVE-11366
[LIVE-11367]: https://ledgerhq.atlassian.net/browse/LIVE-11367

### CODE REVIEW

**Review by jdabbech-ledger — DISMISSED** [review]
lgtm :sparkles:

**Review by valpinkman — APPROVED** [review]

**Review by lduchemin-ledger — APPROVED** [review]
Validation of changes on the e2e tests ✅

File: apps/ledger-live-desktop/src/renderer/hooks/useNftLinks.ts [hunk 1]
```diff
@@ -167,13 +166,13 @@ export default (
...
   }, [account, customImageUri, isInsideDrawer, nft.id, t]);
 
-  const customImageEnabled = useFeature("customImage")?.enabled;
   const devicesModelList = useSelector(devicesModelListSelector);
   const links = useMemo(() => {
     const metadataLinks = linksPerCurrency[account.currency.id]?.(t, metadata?.links) || [];
     return [
       ...metadataLinks,
-      ...(devicesModelList?.includes(DeviceModelId.stax) && customImageEnabled && customImageUri
+      ...(devicesModelList?.find(deviceModelId => isCustomLockScreenSupported(deviceModelId)) &&
```
[the remark this answers is not in the document]
Comment (ofreyssinet-ledger) [reply apps/ledger-live-desktop/src/renderer/hooks/useNftLinks.ts:174]: good point thanks 👍 this still works so can I do it later in another PR so I don't have to go again through the whole QA & PR review process?

File: libs/ledger-live-common/src/device-core/capabilities/isCustomLockScreenSupported.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { DeviceModelId } from "@ledgerhq/devices";
+
+export function isCustomLockScreenSupported(deviceModelId: DeviceModelId) {
+  switch (deviceModelId) {
+    case DeviceModelId.stax:
```
Comment (jiyuzhuang) [inline libs/ledger-live-common/src/device-core/capabilities/isCustomLockScreenSupported.ts:5]: Maybe showing (`return true;`) explicitly the value is a good idea, e.g. if we want to enable / disable one, it's easier and more clear.

**Review by jiyuzhuang — APPROVED** [review]

File: apps/ledger-live-desktop/src/renderer/hooks/useNftLinks.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (beths-ledger) [reply apps/ledger-live-desktop/src/renderer/hooks/useNftLinks.ts:174]: [Aside] this dynamic is my main reservation about the new 'dismiss reviews' protocol on github... (I have found personally it inhibits me from refactoring)

**Review by beths-ledger — APPROVED** [review]

File: apps/ledger-live-desktop/src/renderer/hooks/useNftLinks.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (ofreyssinet-ledger) [reply apps/ledger-live-desktop/src/renderer/hooks/useNftLinks.ts]: @beths-ledger totally agreed

**Review by CremaFR — APPROVED** [review]
LGTM

**Review by jiyuzhuang — APPROVED** [review]

**Review by lduchemin-ledger — APPROVED** [review]

**Review by gre — APPROVED** [review]

**Review by mcayuelas-ledger — APPROVED** [review]

**Pull request merged**